doc/rst/quick.rst

```diff
@@ -143,81 +133,117 @@ Building the firmware
 
     For convenience the release zips provided from XMOS contain precompiled binary (xe) files.
 
-From a command prompt with the XMOS tools available, follow these steps:
+Applications are compiled using `XCommon CMake <https://www.xmos.com/file/xcommon-cmake-documentation/?version=latest>`_ which is a `CMake <https://cmake.org/>`_
+based build system.
 
-    #. Unzip the package zip to a known location
+.. note::
 
-    #. From the relevant application directory (e.g. ``app_usb_aud_xk_audio_316_mc``), execute the commands::
+    See :ref:`proj_build_system` for more details.
 
-        cmake -G "Unix Makefiles" -B build
-        xmake -C build
+Each board is supported by a dedicated application located in its own directory. The boards and
+their corresponding applications are listed in :numref:table_quick_start_boards.
 
-The above steps will configure and build all of the available and supported build configurations for the application.
+.. _table_quick_start_boards:
 
-The applications are compiled using `XCommon CMake <https://www.xmos.com/file/xcommon-cmake-documentation/?version=latest>`_ which is a `CMake <https://cmake.org/>`_
-based build system.
-The primary configuration file for the application is the CMakeLists.txt. It is present in the application directory (e.g. ``app_usb_aud_xk_audio_316_mc``).
+================  ============================
+Board             Application
+================  ============================
+XK-EVK-XU316      app_usb_aud_xk_evk_xu316
+XK-AUDIO-216-MC   app_usb_aud_xk_audio_216_mc
+XK-AUDIO-316-MC   app_usb_aud_xk_audio_316_mc
+================  ============================
+
+The primary configuration for applications is the `CMakeLists.txt`. It is present in each
+application directory (e.g. ``app_usb_aud_xk_audio_316_mc``).
 This file specifies build configs, sources, build options and dependencies.
 
-.. note::
+From a command prompt with the `XMOS` tools available, follow these steps:
 
-    See :ref:`proj_build_system` for more details.
+#. Unzip the package zip to a known location
+
+#. From the relevant application directory (e.g. ``app_usb_aud_xk_audio_316_mc``), execute the commands::
+
+    cmake -G "Unix Makefiles" -B build
+    xmake -C build
```

Comment:
Is -j ignored properly on windows? If so, no harm in keeping it for people running big boy OSes